Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+
+pipeline {
+  agent any
+  stages {
+    stage('build') {
+      when {
+        not {
+          buildingTag()
+        }
+      }
+      steps {
+        sh '/.mounts/labs/gsi/vidarr/jenkins-ci-wrapper test -t /.mounts/labs/gsi/vidarr/testing-config.json'
```

Comment:
@mlaszloffy is the version controlled copy of jenkins-ci-wrapper (https://bitbucket.oicr.on.ca/projects/GSI/repos/infrastructure/browse/gsici.hpc/vidarr) checked out somewhere on the cluster?

Reply:
`/.mounts/labs/gsi/vidarr/jenkins-ci-wrapper` is a symlink to a checkout of git/infra